pkg/controller/toolchainstatus/toolchainstatus_controller_test.go

```diff
@@ -763,6 +763,44 @@ func TestToolchainStatusNotifications(t *testing.T) {
 			// Confirm there is no notification
 			assertToolchainStatusNotificationNotCreated(t, fakeClient)
 
+			t.Run("Notification not created when admin.email not configured", func(t *testing.T) {
+				invalidConfig := &corev1.ConfigMap{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "notification_test_config",
+						Namespace: test.HostOperatorNs,
+					},
+					Data: map[string]string{
+						"admin.email": "",
```

Comment:
Need to add a test for not empty but invalid email.

Reply:
Done